### ISSUE TITLE
router table: at least one broken switch

### ISSUE THREAD

**Issue by LukasFreeze** [comakingspace/do-something#120]
summary:
- footswitch (emergency-off) was discovered to let the machine running about 2/3 of the time
- fix not yet tested: switch on the router and/or table emergency switch malfunction so the router does not turn on in its usual setup (but can be turned on by releasing & pressing the switch on the router that *should* always be pressed)

----

I noticed that the [router table](https://wiki.comakingspace.de/Router_Table) footswitch only works about 1/3 of the time anymore! That can be quite dangerous if you rely on it as an emergency stop...

While the switch is pressed, it seems to work fine, but after releasing it again it can happen that the power returns. This may indicate that the problem actually lies at the table's "loss of power" protection (yellow switchbox at the front) and not the foot pedal itself. We should inspect and clean both switches!

I'd say the equipment can still be used for simple operations as long as you're aware of this, but any blind-ending or otherwise "catchy" work should wait until the problem is resolved and there is a relieable footswitch again.

**Comment by LukasFreeze** [comment]
@lokowandg had a look at the switch and improved grounding (now connected to the case) and strain relief (two cables through one hole, so just a tape hack) but before we could actually test the improvements we noticed the router does not turn on anymore. 

The safety switch on the table is triggered as it should be, and releasing the router's cable tie and pushing the trigger up by hand DID start it just fine. With a new cable tie, we were able to start it one more time and successfully shut it off by foot before it displayed the same behavior again.

In short: the router won't turn on with its trigger on "always on" anymore, any idea what could be the cause?

**Comment by LukasFreeze** [comment]
Die gute Nachricht: alle Schalter funktionieren wie sie sollten!

Die schlechte: Fräse ist hinüber...
![IMG_20190904_212929](https://user-images.githubusercontent.com/19755883/64285047-3df5c000-cf5b-11e9-86ce-bffd158f66fe.jpg)

Eine Reparatur lohnt sich wohl nicht, man könnte aber die Frästisch-Einsatzplatte evtl an die funktionierende Ferm schrauben falls es akuten Bedarf gibt.

**Comment by LukasFreeze** [comment]
Am Tisch ist ein Zettel und das Wiki weiß auch Bescheid, dass jetzt eine andere Fräse mit dem Tisch benutzt werden muss
